### PR TITLE
Add a flag "toolbox.alwaysVisible" to react native

### PR DESCRIPTION
**Pull request by nicolasd**
This pull request adds a flag `toolbox.alwaysVisible` to the react-native application 

This will be used in the `isToolboxVisible` to check if the application should always show the toolbox

### CODE REVIEW

**Comment by nicolasd** [comment]
Already done

File: react/features/base/flags/constants.js [hunk 1]
```diff
@@ -48,3 +48,9 @@ export const PIP_ENABLED = 'pip.enabled';
  * Default: disabled (false).
  */
 export const WELCOME_PAGE_ENABLED = 'welcomepage.enabled';
+
+/**
+ * Flag indicating if the toolbox should be always be visible
+ * Default: disabled (false).
+ */
+export const TOOLBOX_ALWAYS_VISIBLE = 'toolbox.alwaysVisible';
```
Comment (saghul) [inline react/features/base/flags/constants.js:56]: can you please sort this alphabetically?

File: react/features/toolbox/functions.native.js [hunk 2]
```diff
@@ -13,6 +14,7 @@ export function isToolboxVisible(stateful: Object | Function) {
     const state = toState(stateful);
     const { alwaysVisible, enabled, visible } = state['features/toolbox'];
     const { length: participantCount } = state['features/base/participants'];
+    const flag = getFeatureFlag(state, TOOLBOX_ALWAYS_VISIBLE);
```
Comment (saghul) [inline react/features/toolbox/functions.native.js:17]: can you add the `false` default? 

```suggestion
    const flag = getFeatureFlag(state, TOOLBOX_ALWAYS_VISIBLE, false);
```

File: react/features/base/flags/constants.js (hunk 1 above)
Comment (nicolasd) [reply react/features/base/flags/constants.js]: Thanks, it's done

File: react/features/toolbox/functions.native.js (hunk 2 above)
Comment (nicolasd) [reply react/features/toolbox/functions.native.js]: Done

**Comment by nicolasd** [comment]
Hello, i made all of your requested change

**Pull request merged**